ci/before_build.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue'
+
+# https://github.com/lukesampson/scoop#installation
+Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')
+
+Set-StrictMode -Version Latest
```

Comment:
If this doesn't work, scoop can't be used in the CI without disabling strict mode.

Reply:
It works.